server/seeds/demoSeed.js

```diff
@@ -0,0 +1,26 @@
+const dotenv = require('dotenv').config({path:__dirname+'/./../.env'})
+const mongoose = require("mongoose");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+
+const seedDB = async (req, res, next) => {
+  const salt = await bcrypt.genSalt(10);
+  saltedPassword = await bcrypt.hash(process.env.DEMO_USER_PASSWORD, salt);
+  
+  const demoUser = [
+    {
+      email: process.env.DEMO_USER_EMAIL,
+      password: saltedPassword,
+      name: "demo",
+    },
+  ]
```

Comment:
Is there a particular reason you went with an array here? We should only ever really have on demo user setup at a given time.

Reply:
updated - thanks! no particular reason here - that was a miss on my part.